boilerplate/app/components/Screen.tsx

```diff
@@ -265,19 +265,28 @@ export function Screen(props: ScreenProps) {
         style={statusBarStyle || (themeContext === "dark" ? "light" : "dark")}
         {...StatusBarProps}
       />
-
-      <KeyboardAvoidingView
-        behavior={isIos ? "padding" : "height"}
-        keyboardVerticalOffset={keyboardOffset}
-        {...KeyboardAvoidingViewProps}
-        style={[$styles.flex1, KeyboardAvoidingViewProps?.style]}
-      >
-        {isNonScrolling(props.preset) ? (
+      {/**
+       * KeyboardAvoidingView crashes in web,
+       * therefore we want to use ScrollView just for web
+       */}
+      {Platform.OS === "web" ? (
+        <ScrollView>
           <ScreenWithoutScrolling {...props} />
```

Comment:
Also meant to ask: why does web only get `ScreeenWithoutScrolling`? Shouldn't it have both options?

I think that's why this appeared to fix things, it removes `KeyboardAwareScrollView` from the document.